src/Uno.UI/UI/Xaml/UIElement.Pointers.Skia.cs

```diff
@@ -0,0 +1,328 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using Windows.Devices.Input;
+using Windows.UI.Input;
+using Windows.UI.Xaml.Input;
+using Microsoft.Extensions.Logging;
+using Uno.Extensions;
+using Uno.Logging;
+using Uno.Foundation.Extensibility;
+using Windows.UI.Core;
+using Windows.Foundation;
+
+namespace Windows.UI.Xaml
+{
+	partial class UIElement
+	{
+		private class PointerManager
+		{
+			public PointerManager()
+			{
+				Window.Current.CoreWindow.PointerMoved += CoreWindow_PointerMoved;
+				Window.Current.CoreWindow.PointerEntered += CoreWindow_PointerEntered;
+				Window.Current.CoreWindow.PointerExited += CoreWindow_PointerExited;
+				Window.Current.CoreWindow.PointerPressed += CoreWindow_PointerPressed;
+				Window.Current.CoreWindow.PointerReleased += CoreWindow_PointerReleased;
+				Window.Current.CoreWindow.PointerWheelChanged += CoreWindow_PointerWheelChanged;
+			}
+
+			private void CoreWindow_PointerWheelChanged(CoreWindow sender, PointerEventArgs args)
+			{
+				if (this.Log().IsEnabled(LogLevel.Trace))
+				{
+					this.Log().Trace($"CoreWindow_PointerWheelChanged ({args.CurrentPoint.Position})");
+				}
+
+				PropagateEvent(args, e =>
+				{
+					var pointer = new Pointer(args.CurrentPoint.PointerId, PointerDeviceType.Mouse, false, isInRange: true);
+					var pointerArgs = new PointerRoutedEventArgs(args, pointer, e) { CanBubbleNatively = true };
+					TraverseAncestors(e, pointerArgs, e2 => e2.OnNativePointerWheel(pointerArgs));
+				});
+			}
+
+			private void CoreWindow_PointerEntered(CoreWindow sender, PointerEventArgs args)
+			{
+				if (this.Log().IsEnabled(LogLevel.Trace))
+				{
+					this.Log().Trace($"CoreWindow_PointerEntered ({args.CurrentPoint.Position})");
+				}
+			}
+
+			private void CoreWindow_PointerExited(CoreWindow sender, PointerEventArgs args)
+			{
+				if (this.Log().IsEnabled(LogLevel.Trace))
+				{
+					this.Log().Trace($"CoreWindow_PointerExited ({args.CurrentPoint.Position})");
+				}
+
+
+			}
+
+			private void CoreWindow_PointerReleased(CoreWindow sender, PointerEventArgs args)
+			{
+				if (this.Log().IsEnabled(LogLevel.Trace))
+				{
+					this.Log().Trace($"CoreWindow_PointerReleased ({args.CurrentPoint.Position})");
+				}
+
+				var pointer = new Pointer(args.CurrentPoint.PointerId, PointerDeviceType.Mouse, false, isInRange: true);
+				if (UIElement.PointerCapture.TryGet(pointer, out var capture))
+				{
+					foreach(var target in capture.Targets)
+					{
+						var pointerArgs = new PointerRoutedEventArgs(args, pointer, target.Element) { CanBubbleNatively = true };
+						TraverseAncestors(target.Element, pointerArgs, e => e.OnNativePointerUp(pointerArgs));
+					}
+				}
+				else
+				{
+					PropagateEvent(args, e =>
+					{
+						// Console.WriteLine($"PointerManager.Released [{e}/{e.GetHashCode():X8}");
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5872302)